src/lang/Language.php

```diff
@@ -59,7 +62,24 @@ public static function getLanguageList(string $path = "") : array{
 			$path = \pocketmine\LOCALE_DATA_PATH;
 		}
 
-		if(is_dir($path)){
+		if(str_contains($path, ".phar/")){
```

Comment:
phar paths should be distinguished by checking `str_starts_with($path, "phar://")`, right?